filter/patterns_storage.go

```diff
@@ -62,6 +66,11 @@ func (storage *PatternStorage) ProcessIncomingMetric(lineBytes []byte) *moira.Ma
 		return nil
 	}
 
+	if parsedMetric.IsTooOld(maxTTL, storage.clock.Now().UTC()) {
```

Comment:
Let's use our new SystemClock without UTC conversion again

Reply:
done, furthermore, use helper func Int64ToTime - return datetime from timestamp in UTC.